btree/btree.c

```diff
@@ -2332,8 +2332,8 @@ static void mem_tree_put(struct td *tree)
 		m0_rwlock_write_lock(&trees_lock);
 		M0_ASSERT(trees_loaded > 0);
 		i = tree - &trees[0];
-		array_offset = i / sizeof(trees_in_use[0]);
-		bit_offset_in_array = i % sizeof(trees_in_use[0]);
+		array_offset = i / (sizeof(trees_in_use[0]) * 8);
+		bit_offset_in_array = i % (sizeof(trees_in_use[0]) * 8);
```

Comment:
8 is hardcoded can we replace it by defining cost variable and use it here

Reply:
Multiplying by 8 to convert it to bits. Anyways, this code will be replaced after tree_get/put changes.

Reply:
okay